content/departments/engineering/linear-trial.md

```diff
@@ -0,0 +1,234 @@
+   - Sync issues after import
+
+   So that syncing is maintained for old issues, and all issue history
+   is available for search.
+
+   After this step, select 'Import users' and make sure that:
+
+   - If a person has 'Map to Existing' but no associated account on
+     the right side, then change it to 'Do not create'
+   - Replace any 'Create with email' with 'Do not create'
+
+   Once you click next, confirm on the next page that:
+
+   - The number of user invites is zero
+   - The number of completed issues matches up with the closed
+     issues in the repo's GitHub issue tracker.
+
+At the moment, all repos except for the monorepo use an extra issue label
+to allow filtering for views inside Linear. For example, the `scip-java`
+repo uses the `graph/scip-java` label.
+Here is an [example PR](https://github.com/sourcegraph/scip-java/pull/685/files)
+for auto-adding the label to new issues.
+For old issues, you can quickly add the label after importing into Linear.
+
+## Dos and Don'ts
+
+1.  **Don't create new Teams for sub-teams in Engineering**
+    (except for testing Linear features, that's OK).
+    Instead, let's try to tweak existing settings or enable new ones
+    which would be useful to all teams.
+2.  **Do add other repositories of interest to Linear**.
+    - Even for repos which only have PRs, if the PR descriptions
+      use Linear identifiers such as `Fixes ENG-123`,
+      then the corresponding issue will be auto-closed.
+    - Caveat: Do not enable bidirectional syncing for other repos
+      in Eng PUBLIC or Eng Private.
+3.  If your team is on-board, **do connect your team's #discuss
+    channel with Eng Private using Linear Asks**
+    ([here](https://linear.app/sourcegraph/settings/asks))
+    to easily turn Slack threads into issues. ([docs](https://linear.app/docs/linear-asks))
+4.  **Avoid creating new label groups inside Linear**
+    (creating new labels, is OK). - If you attempt to create a new label with `/` on-the-fly
+    when editing an issue, Linear will create a label group
+    instead of a "flat" label.
+
+    For example, if you type `X/Y` for a new label, it will create
+    a group `X` with a label `Y` in that group. Since `X` is the
+    group name, the label synced to GitHub will be just `Y`.
+    This may be confusing to someone seeing the issue on GitHub
+    as the prefix is no longer available.
+
+    - Label groups also have at-most-one semantics - you can't apply
+      multiple labels from the same group to a single issue.
+    - Alternative: you can create a label with `/` from the Team's settings
+      ([example link](https://linear.app/sourcegraph/settings/teams/ENG/labels)).
+
+5.  **Do explore the various Linear [features](https://linear.app/docs)
+    and [integrations](https://linear.app/sourcegraph/settings/integrations)**.
+
+## FAQ
+
+### Why are Eng PUBLIC and Eng Private separate?
+
+The Eng Private team is present because of the following
```

Comment:
Views in Linear are not necessarily tied to Teams, so that shouldn't be a problem. I noticed that I mistakenly made the Graph team issues view to be scoped to the Eng PUBLIC team, but you can see the [new View](https://linear.app/sourcegraph/view/7b4400f7-6e98-4847-b628-6645871c0621) which contains issues from both Eng PUBLIC and Eng Private

Reply:
oh that is nice, views seem super flexible